SimG4Core/Application/src/RunManagerMTWorker.cc

```diff
@@ -340,6 +342,7 @@ void RunManagerMTWorker::initializeG4(RunManagerMT* runManagerMaster, const edm:
 
   // Set the physics list for the worker, share from master
   PhysicsList* physicsList = runManagerMaster->physicsListForWorker();
+  m_isPhase2 = runManagerMaster->isPhase2();
```

Comment:
There is probably no need to make this variable a class member of RunManagerMTWorker, it could probably better stay local. On the other hand, it is done as such also or the other `m_*` boolean class member, and perhaps there was a good reason to do so...